fbu-app/PostSearch/PostSearchLeftViewController/PostSearchLeftViewController.m

```diff
@@ -30,18 +30,10 @@ - (void)viewDidLoad {
     [self.tableView setBackgroundColor:[UIColor clearColor]];
     self.tableView.dataSource = self;
     self.tableView.delegate = self;
-    titlesArray = @[@" ",
-                    @"Roommate Finder",
+    titlesArray = @[@"Roommate Finder",
+                    @"Received Requests",
+                    @"Sent Requests",
```

Comment:
for future reference whenever you change this your going to have to change the switch statement